src/dlg_audiooptions.cpp

```diff
@@ -119,7 +119,7 @@ AudioOptsDialog::AudioOptsDialog(wxWindow* parent, wxWindowID id, const wxString
     // Rx Out -----------------------------------------------------------------------
 
     wxStaticBoxSizer* sbSizer3;
-    sbSizer3 = new wxStaticBoxSizer(new wxStaticBox(m_panelRx, wxID_ANY, _("From Computer To Speaker/Headphones")), wxHORIZONTAL);
+    sbSizer3 = new wxStaticBoxSizer(new wxStaticBox(m_panelRx, wxID_ANY, _("Output From Computer To Speaker/Headphones")), wxHORIZONTAL);
 
```

Comment:
Interesting innovation :thinking: - thanks for also modifying the user manual to match!  I'm open to ideas here, TBH I'm not sure how best the convey this issue to new users.  It's certainly a source of confusion.

Reply:
I figured explicitly mentioning where audio flows from towards would help clarify that a bit. Maybe longer term there can be a "simple" mode where you can select which sound card the radio is plugged into/which one has your headset and FreeDV can figure it out for you? Something like that's outside the scope of this PR though.

Reply:
Yep there is much that could be done here. A default Rx setup that uses the default sound device would be useful to get people started. 

About 12 months ago @ac0zj had a go at [filtering](https://github.com/drowe67/freedv-gui/pull/24) the long lists of sound devices but it was difficult to get consistent behavior across operating systems.

Reply:
Yes another suggestion for a simple GUI mode is a way to "disappear" many of the options to make it less bewildering for noobs.